src/background/controllers/accountController.ts

```diff
@@ -436,6 +439,18 @@ export default class AccountController extends IController {
     chrome.runtime.sendMessage({ type: MESSAGE_TYPE.UNEXPECTED_ERROR, error: err.message });
   }
 
+  private sendMaxQtumSendToPopup = async () => {
+    if (!this.loggedInAccount || !this.loggedInAccount.wallet || !this.loggedInAccount.wallet.qjsWallet) {
+      throw Error('Cannot calculate max balance with no wallet instance.');
+    }
+
+    const mqsaPromise = this.loggedInAccount.wallet.calcMaxQtumSend(this.main.network.networkName);
+    mqsaPromise.then((maxQtumSend) => {
```

Comment:
-It doesn't make a huge difference, but if I change it to async/await it will make this function call of this.sendMaxQtumAmountToPopup() below at Line 369 blocking right?
-And if it is blocking I believe the chrome.runtime.sendMessage  at the bottom of the method would fire a little bit later right?

private getWalletInfo = async (sendInpageUpdate = true) => {
    if (!this.loggedInAccount || !this.loggedInAccount.wallet || !this.loggedInAccount.wallet.qjsWallet) {
      console.error('Could not get wallet info.');
      return;
    }

    let existingBalance;
    if (this.loggedInAccount.wallet.info) {
      existingBalance = this.loggedInAccount.wallet.info!.balance;
    }

    await this.loggedInAccount.wallet.getInfo();
    const newBalance = this.loggedInAccount.wallet.info!.balance;

    if (existingBalance !== newBalance) {
      if (sendInpageUpdate) {
        this.main.inpageAccount.sendInpageAccountAllPorts(QRYPTO_ACCOUNT_CHANGE.BALANCE_CHANGE);
      }
      this.sendMaxQtumAmountToPopup();
    }

    chrome.runtime.sendMessage({ type: MESSAGE_TYPE.GET_WALLET_INFO_RETURN, info: this.loggedInAccount.wallet.info });
  }

Reply:
```
promiseFunc1().then(() => {
  func2();
});
```
is equivalent to:
```
await promiseFunc1();
func2();
```